### PR TITLE
Add README for arduino directory

### LINKED ISSUE #2: Create directory for Arduino code
We need a place to store our various Arduino solutions. These should be gathered together in a separate directory under the repo root.

### PR DESCRIPTION
**Pull request by cariehl**

### CODE REVIEW

File: arduino/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 2024-Robot\arduino
+
```
Comment (OverlandCrib959) [inline arduino/README.md:2]: yes

File: arduino/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+This directory contains the various Arduino projects we are using/investigating
```
Comment (OverlandCrib959) [inline arduino/README.md:3]: indeed

File: arduino/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+for use with our 2024 robot. Please create a directory for each project, i.e.
+'arduino\leds' or 'arudino\sensors\TOF400F'
```
Comment (CH-FRC3767) [inline arduino/README.md:5]: This is line 5

**Review by CH-FRC3767 — APPROVED** [review]

**Review by OverlandCrib959 — APPROVED** [review]
e

File: arduino/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# 2024-Robot\arduino
```
Comment (PrestonHalverson) [inline arduino/README.md:1]: Robot🔥🔥🎉🎉🎉📢📢

**Review by PrestonHalverson — APPROVED** [review]

**Pull request merged**